lib/rubocop/cop/style/documentation.rb

```diff
@@ -31,6 +32,9 @@ class Documentation < Cop
 
         def_node_matcher :constant_definition?, '{class module casgn}'
         def_node_search :outer_module, '(const (const nil? _) _)'
+        def_node_matcher :constant_visibility_declaration?, <<-PATTERN
```

Comment:
Can you rebase with latest master branch and use squiggly heredoc?

```diff
- def_node_matcher :constant_visibility_declaration?, <<-PATTERN
+ def_node_matcher :constant_visibility_declaration?, <<~PATTERN
```

Reply:
@koic done :)